libs/librepcb/common/dialogs/aboutdialog.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * LibrePCB - Professional EDA for everyone!
+ * Copyright (C) 2018 LibrePCB Developers, see AUTHORS.md for contributors.
+ * http://librepcb.org/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*****************************************************************************************
+ *  Includes
+ ****************************************************************************************/
+#include "aboutdialog.h"
+#include "ui_aboutdialog.h"
+#include <librepcb/common/application.h>
+
+/*****************************************************************************************
+ *  Namespace
+ ****************************************************************************************/
+namespace librepcb {
+
+AboutDialog::AboutDialog(QWidget* parent) noexcept :
+    QDialog(parent),
+    mUi(new Ui::AboutDialog)
+{
+    // Set up base dialog
+    mUi->setupUi(this);
+
+    // Event handlers
+    connect(mUi->buttonBox, &QDialogButtonBox::clicked,
+            this, &AboutDialog::close);
+
+    // Layout
+    mUi->tabWidget->setCurrentIndex(0);
+
+    // Get some version information
+    const Version& appVersion = qApp->getAppVersion();
+    const QString& gitVersion = qApp->getGitVersion();
+    const QString& buildDate = qApp->getBuildDate().toString("yyyy-MM-dd hh:mm:ss (t)");
+
+    // Dynamic text
+    mUi->textVersion->setText(QString("%1 (%2)<br>Build date: %3").arg(appVersion.toPrettyStr(3), gitVersion, buildDate));
+    mUi->textLinks->setText(tr("For more information, please check out <a href='%1'>librepcb.org</a><br>or our <a href='%2'>GitHub repository</a>.").arg("http://librepcb.org/", "https://github.com/LibrePCB/LibrePCB"));
```

Comment:
Could you remove the `<br>` in this label? All other labels also don't have manual line breaks.